plasmapy/utils/datatype_factory_base.py

```diff
@@ -160,7 +163,7 @@ def register(self, WidgetType, validation_function=None, is_default=False):
                 if hasattr(WidgetType, vfunc_str):
                     vfunc = getattr(WidgetType, vfunc_str)
 
-                    # check if classmethod: stackoverflow #19227724
+                    # check if classmethod: stackoverflow #19227g724
```

Comment:
```suggestion
                    # check if classmethod: stackoverflow #19227724
```
I think an extra character slipped in here.